docs/develop/api-reference/response-objects.md

```diff
@@ -38,4 +38,12 @@ For fetch commands, the key or field exists in the cache and there is a "cache h
 
 ### Miss
 
-For fetch commands, the key or field does not exist and it a "cache miss."
\ No newline at end of file
+For fetch commands, the key or field does not exist and it a "cache miss."
+
+### Stored
+
+For setIfNotExists commands, the key did not exist and the value was set.
```

Comment:
I'm starting to think this is going to grow to a point where it makes sense to just keep the definitions with the descriptions. No need to link out to a response-objects page.
Thoughts @NoSQLKnowHow @pete-naylor ?